src/webui/www/private/views/preferences.html

```diff
@@ -2551,6 +2573,16 @@
             // Validate form data
 
             // Behavior tab
+            // Language
+            settings["locale"] = $("locale_select").value;
+            const colorScheme = Number(document.getElementById("colorSchemeSelect").value);
+            if (colorScheme === 0)
+                LocalPreferences.remove("color_scheme");
+            else if (colorScheme === 1)
+                LocalPreferences.set("color_scheme", "light");
```

Comment:
Should I use camelCase for the settings' key too?

Reply:
I don't think we have rules for it. Snake case is fine for now.